CMakeLists.txt

```diff
@@ -279,6 +279,10 @@ endif()
 if(GINKGO_BUILD_HWLOC)
     find_package(HWLOC 2.1) # No need for QUIET as we ship FindHWLOC
 endif()
+if(GINKGO_WITH_PAPI_SDE)
+    # No need for QUIET as we ship FindPAPI
+    find_package(PAPI 7.0.2 OPTIONAL_COMPONENTS sde)
```

Comment:
We require sde from papi, right?
putting it in optional components will still mark the PAPI found even if the PAPI does not contain sde.